src/app/application.cpp

```diff
@@ -171,6 +173,7 @@ Application::Application(const QString &id, int &argc, char **argv)
         m_fileLogger = new FileLogger(fileLoggerPath(), isFileLoggerBackup(), fileLoggerMaxSize(), isFileLoggerDeleteOld(), fileLoggerAge(), static_cast<FileLogger::FileLogAgeType>(fileLoggerAgeType()));
 
     Logger::instance()->addMessage(tr("qBittorrent %1 started", "qBittorrent v3.2.0alpha started").arg(QBT_VERSION));
+    Logger::instance()->addMessage(tr("Using config directory: %1").arg(Profile::instance().location(SpecialFolder::Config)));
```

Comment:
I would add
```c++
if (portableModeEnabled)
    LogMsg(tr("Running in portable mode (auto detected)"));
```
Also I would move appropriate check from "validateCommandLineParameters" here (since it doesn't really validate command-line parameters themselves) so log messages will in correct order ("portable mode enabled" and then "relative fastresume is implied"). Also we will have no additional member variable.

Reply:
Moved check from "validateCommandLineParameters" here added additional log message. Please let me know if I should change anything else.